_XOPEN_SOURCE 700 is unknown in 11.3; use _XOPEN_SOURCE 600/XPG6 instead

--- a/meson.build	2023-03-30 18:26:30.397462433 +0000
+++ b/meson.build	2023-03-30 18:27:12.080556688 +0000
@@ -164,8 +164,7 @@
 libc_feature_defines = [
   ['_GNU_SOURCE', '1'],
   ['_POSIX_C_SOURCE', '200809L'],
-  ['_XOPEN_SOURCE', '700'],
-  ['_XOPEN_SOURCE_EXTENDED', '1'],
+  ['_XOPEN_SOURCE', '600'],
 ]
 
 system = host_machine.system()
